internal/archive/archive.go

```diff
@@ -80,6 +81,12 @@ func openUbuntu(options *Options) (Archive, error) {
 		return nil, fmt.Errorf("no data about Ubuntu version %s", options.Version)
 	}
 
+	suites := []string{animal}
+	suite_suffixes := []string{"updates", "security"}
+	for _, suite_suffix := range suite_suffixes {
+		suites = append(suites, animal+"-"+suite_suffix)
+	}
```

Comment:
:+1: we'll add the test for now, and make a reference to this in #20 where the archive's configuration is supposed to be improved

Reply:
This would be solved by https://github.com/canonical/chisel/issues/20 @niemeyer What do you think about that approach? It mirrors how Apt sources are specified. (Commented the same at the bottom.)